lib/Dialect/TritonNvidiaGPU/IR/Ops.cpp

```diff
@@ -277,6 +277,7 @@ static std::string strMMADTypeKind(MMADTypeKind kind) {
   case MMADTypeKind::i8:
     return "i8";
   }
+  return failure();
```

Comment:
The return type doesn't match. Why not use something like `llvm_unreachable`

Reply:
Sounds great I changed it to `__builtin_unreachable`

Reply:
We have included llvm headers. It does not matter if the compiler is GCC or CLANG